_posts/2019/2019-05-21-making-a-move-abroad-from-bolivia-to-hamburg.md

```diff
@@ -0,0 +1,58 @@
+---
+layout: post
+title: Making a move abroad - from Bolivia to Hamburg
+date: 2019-05-21
+header_image: public/relocation.jpg
+header_position: center
+header_overlay: true
+category: on-the-job
+tags: ["onboarding", "relocation", "new job"]
+authors: ["Kathia"]
+about_authors: ["ksalazar"]
+---
+
+Hi, my name is Kathia. 
+I come from Bolivia, and since March 2019 I'm working as a Software Developer at ePages in Hamburg.
+In late 2018 I applied online, and then had a first interview with Serena, the HR recruiter.
```

Comment:
Did you have all the interviews via Hangouts? Or have you visited the office once to get to know the people? How was that for you? Maybe you can shed some light on this aspect.